README.md

```diff
@@ -179,6 +179,10 @@ a suggestion is attempted to be highlighted either by the keyboard or mouse. It
 scope, access to an object called `suggestion` which is the current suggestion that is being
 interacted with. A non-selectable suggestion cannot be clicked on, hovered over, or interacted with
 via the keyboard.
+ - `inputHint({query}) {Promise}`: An expression that hints at a suggestion by resolving a
```

Comment:
But they can't be resolved separately. The hinting will always have to wait for the suggestions to finish loading, even if it's a much heavier, slower call, and even if the hints are available locally.